src/bot/config.ts

```diff
@@ -23,25 +23,13 @@ export const getGitHubConfig = async (orgId: string) => {
 
   const orgData = await octokit.rest.orgs.get({ org: orgId })
 
-  const config = await octokit.config.get<InternalContributionForksConfig>({
-    owner: orgData.data.login,
-    repo: '.github',
-    path: 'internal-contribution-forks/config.yml',
-  })
-
-  const found = Boolean(config.files[0].config)
-
-  if (!found) {
-    configLogger.warn(
-      `No config found for org, using default org: '${orgId}' for BOTH public and private!`,
-    )
-    return {
-      publicOrg: orgId,
-      privateOrg: orgId,
-    }
+  configLogger.warn(
```

Comment:
Do we still need this log since we are only using the default config now?

Reply:
Good point, let me update